app/assets/stylesheets/application.css.erb

```diff
@@ -1962,3 +1962,9 @@ input[type="submit"].link_post.pushover_button {
 		float: none;
 	}
 }
+@media print {
+	.comment_folder,
+	.comment_form {
+	    display: none;
```

Comment:
Thanks for taking this on.

I hate to bring up tabs vs spaces but we don't have linting on css. Could you make this single-tab per indent level to match the file?

Reply:
ok let me change it to tab per ident